### DISCUSSION TITLE
Gitter

### DISCUSSION THREAD

**Discussion by keradus** [PHP-CS-Fixer/PHP-CS-Fixer#7211] [RFC]
also cc @localheinz @julienfalque @kubawerlos @SpacePossum @Wirone for comments

**Comment by kubawerlos** [comment]
Since the Gitter's migration to Element I had a lot of trouble login with the same account (AFAIR there was a different account when I logged in with an email, different when I logged in via Google, and different when via GitHub), so I gave up with that s**t.

**Reply by Wirone** [reply]
@kubawerlos I created one matrix.org account and don't have any problems with it. I have active session in the browser and mobile app installed. Element is working correctly 🤷‍♂️.

**Comment by Wirone** [comment]
@keradus the question is not specific enough - if you're talking about support channel then YES, we should sunset Gitter and focus on Github Discussions. But if you mean getting rid of chat completely (not only public lobby) then NO, we need some way of communicating without notification overhead.

**Reply by keradus** [reply]
In this Discussion I created on repo, I have in mind only the public communication channel for anyone eager to discuss or ask around Fixer.

for internal communication of core team - we should decide on that, but not the scope of my original question (and I think we can take it to private discussion)

**Reply by Wirone** [reply]
Then as I said before, I think we should sunset Gitter and focus on Github Discussions, because:
- it can easily work as FAQ
- it's has better search-ability (directly on Github, but also from Google)
- it's easier to maintain community's problems/questions in one place

**Reply by keradus** [reply]
I think we are concluded then.

**Reply by Wirone** [reply]
@keradus I've created [PR](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7255) that follows this RFC.

**Comment by keradus** [comment]
Conclusion:
- sunset gitter for public support, use GitHub discussions
